tests/unit/utils/containers.go

```diff
@@ -0,0 +1,48 @@
+package utils
+
+import (
+	cassdcV1beta1 "github.com/datastax/cass-operator/operator/pkg/apis/cassandra/v1beta1"
+	. "github.com/onsi/gomega"
+	coreV1 "k8s.io/api/core/v1"
+)
+
+func GetInitContainer(cassdc *cassdcV1beta1.CassandraDatacenter, name string) *coreV1.Container {
```

Comment:
Can you change the name to `GetCassdcInitContainer`? You cannot have overloaded functions in Go :(